### PR TITLE
Add Performance

### PR DESCRIPTION
**Pull request by kaycebasques**

### CODE REVIEW

**Comment by kaycebasques** [comment]
The error output of the linter isn't very helpful...

**Comment by caugner** [comment]
> The error output of the linter isn't very helpful...

Just to let you know that this was meanwhile fixed in #2136.

File: api/Performance.json [hunk 1]
```diff
@@ -50,48 +48,440 @@
...
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "memory": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Performance/memory",
+          "support": {},
```
Comment (caugner) [inline api/Performance.json]: I have just manually checked that it is:
* supported by Chrome and Opera,
* not supported by Firefox.

Please also add all other browsers used in the `Performance.__compat` section and set them to `null`.

File: api/Performance.json [hunk 2]
```diff
@@ -50,48 +48,440 @@
...
+          "status": {
+            "experimental": false,
+            "standard_track": true,
```
Comment (caugner) [inline api/Performance.json]: `Performance.memory` is a non-standard extension according to the [MDN article](https://developer.mozilla.org/en-US/docs/Web/API/Performance#Properties).

File: api/Performance.json [hunk 3]
```diff
@@ -106,43 +496,223 @@
...
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Performance/onresourcetimingbufferfull",
+          "support": {
+            "webview_android": {
+              "version_added": true
+            },
+            "chrome": {
+              "version_added": true
+            },
+            "chrome_android": {
+              "version_added": true
```
Comment (caugner) [inline api/Performance.json]: The [MDN article for this method](https://developer.mozilla.org/en-US/docs/Web/API/Performance/onresourcetimingbufferfull#Browser_compatibility) has more information, see above.

File: api/Performance.json [hunk 4]
```diff
@@ -106,43 +496,223 @@
...
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Performance/setResourceTimingBufferSize",
+          "support": {
+            "webview_android": {
+              "version_added": true
+            },
+            "chrome": {
+              "version_added": true
+            },
+            "chrome_android": {
+              "version_added": true
```
Comment (caugner) [inline api/Performance.json]: The [MDN article for this method](https://developer.mozilla.org/en-US/docs/Web/API/Performance/setResourceTimingBufferSize#Browser_compatibility) has more information, see above.

File: api/Performance.json [hunk 5]
```diff
@@ -157,43 +727,40 @@
...
             "firefox_android": {
-              "version_added": null
+              "version_added": "25"
             },
             "ie": {
               "version_added": null
             },
             "opera": {
-              "version_added": null
+              "version_added": false
```
Comment (caugner) [inline api/Performance.json]: See above.

File: api/Performance.json [hunk 6]
```diff
@@ -106,43 +496,223 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/Performance/now",
           "support": {
             "webview_android": {
-              "version_added": null
+              "version_added": "25"
             },
             "chrome": {
-              "version_added": null
+              "version_added": "24",
+              "notes": "Windows versions of Chrome 20 through 33 return <code>performance.now()</code> only to millisecond precision."
```
[the remark this answers is not in the document]
Comment (Elchi3) [reply api/Performance.json]: I've removed this note, as I think it makes no sense anymore now that browsers have no millisecond precision anymore due to spectre and meltdown

File: api/Performance.json [hunk 7]
```diff
@@ -5,43 +5,41 @@
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/Performance",
         "support": {
           "webview_android": {
-            "version_added": null
+            "version_added": true
           },
           "chrome": {
-            "version_added": null
+            "version_added": "6",
+            "notes": "Member prefixed with webkit have been removed or deprecated."
```
[the remark this answers is not in the document]
Comment (Elchi3) [reply api/Performance.json]: This note makes no sense to me. I removed it.

File: api/Performance.json [hunk 8]
```diff
@@ -50,48 +48,440 @@
...
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Performance/clearResourceTimings",
+          "support": {
+            "webview_android": {
+              "version_added": true
+            },
+            "chrome": {
+              "version_added": true
+            },
+            "chrome_android": {
+              "version_added": true
```
[the remark this answers is not in the document]
Comment (Elchi3) [reply api/Performance.json]: Added

**Comment by Elchi3** [comment]
I've updated this. Ready for re-review.

**Comment by ddbeck** [comment]
Let's merge this! 🎉

**Pull request merged**